.circleci/config.yml

```
@@ -150,6 +150,7 @@ workflows:
                 - "3.7.9"
                 - "3.8.13"
                 - "3.9.1"
+                - "3.10.6"
```

I think we should use the final available version 3.10.9

```suggestion
                - "3.10.9"
```

Ref: https://docs.python.org/3.10/whatsnew/changelog.html#python-3-10-9-final